forge-gui/res/tokenscripts/c_2_2_a_drone_deathtouch_leavedrain.txt

```diff
@@ -0,0 +1,9 @@
+Name:Drone Token
+ManaCost:no cost
+Types:Artifact Creature Drone
+PT:2/2
+K:Deathtouch
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Execute$ TrigDrain | TriggerDescription$ When this creature leaves the battlefield, each opponent loses 2 life and you gain 2 life.
```

Comment:
I think this should have `Destination$ Any`

Reply:
Okay, no problem to add if convention calls for it. In testing, this seems to work functionally the same, as this line was taken from Circuit Mender's script.

Reply:
No, you're right, it isn't needed. Either way is fine ;)